packages/web-components/fast-foundation/src/slider/slider.ts

```diff
@@ -408,6 +411,12 @@ export class Slider extends FormAssociated<HTMLInputElement>
                     ? e.pageX - this.getBoundingClientRect().left
                     : e.pageY;
 
+            if (this.orientation === Orientation.vertical) {
+                const clientRect: DOMRect = this.track.getBoundingClientRect();
```

Comment:
getBoundingClientRect is an expensive call, you can't just get the clientHeight from track element directly?  I see we also do it on mousemove which is concerning because it can get called a lot.

Reply:
I believe we had something else that wasn't getBoundingClientRect before and it hosed up some other situation. I'll give clientHeight a shot and see if it'll work in at least the fixtures we have now though

Reply:
Resize observer might be another option.  I'd consider calling getBoundingClientRect on a high frequency event like mousemove a perf red flag.

Reply:
Thanks to a suggestion from Jane, I'm only doing the getClientBoundingRect in mouseDown and holding on to that for the duration of mousemove, so I believe this should alleviate the perf concern, also fixes the bug.